robot/basestation/ROS3D.md

```diff
@@ -0,0 +1,15 @@
+# Displaying the robot in the GUI
+
+## Display the Arm
+Upon starting the GUI with `rosgui` and then `startgui`, these are the steps to see the arm.
+
+1) Open a terminal with `venv` activated and run `roslaunch arm_vision.launch`
```

Comment:
this should be `roslaunch astro_arm arm_vision.launch`, same for the line below it